example/src/main/java/com/iopipe/examples/Hello.java

```diff
@@ -56,6 +56,9 @@ protected final String wrappedHandleRequest(IOpipeExecution __exec,
 			// Store the result of the math
 			__exec.customMetric("result", (long)result);
 		}
+		
+		// Test label
+		__exec.label("test");
```

Comment:
I love these additions, but because of reason mentioned in pom.xml, they'll need to be added with the 1.2 release. Will think more on this limitation.

Reply:
Reverted changes.